octobot_trading/personal_data/trades/trades_util.py

```diff
@@ -66,3 +66,25 @@ def compute_win_rate(exchange_manager):
     if total_counted_trades > constants.ZERO:
         return won_trades_count / total_counted_trades
     return constants.ZERO
+
+
+def aggregate_trades_by_exchange_order_id(trades: list) -> dict:
+    aggregated_trades_by_exchange_order_id = {}
+    for trade in trades:
+        if trade.exchange_order_id not in aggregated_trades_by_exchange_order_id:
+            # initialise as a new trade that will contain the aggregation
+            aggregated_trades_by_exchange_order_id[trade.exchange_order_id] = personal_data.create_trade_from_dict(
+                trade.trader, trade.to_dict()
+            )
+        else:
+            # include other trades from the same order id into the aggregated trade
+            to_update = aggregated_trades_by_exchange_order_id[trade.exchange_order_id]
+            to_update.executed_price = (
+                to_update.executed_quantity * to_update.executed_price + trade.executed_quantity * trade.executed_price
+            ) / (to_update.executed_quantity + trade.executed_quantity)
```

Comment:
I think we should catch some calculation issue here like DivideByZero

Reply:
indeed, I updated it, thanks